build.sbt

```diff
@@ -533,6 +534,8 @@ lazy val componentModulesPaths =
     (`zio-wrapper` / Compile / exportedModuleBin).value,
     (`language-server-deps-wrapper` / Compile / exportedModuleBin).value,
     (`ydoc-polyfill` / Compile / exportedModuleBin).value,
+    (`ydoc-server` / Compile / exportedModuleBin).value,
```

Comment:
Because now ydoc-server is a dependency of `buildEngineDistribution`, it started failing bazel build. The correct fix is: 
- generate ydoc-server using bazel
- provide it for sbt as artifact
- remove Ydoc.scala

Unfortunately, I’m not versed enough with sbt, @Akirathan should be able to help.

Reply:
> * generate ydoc-server using bazel

How that would be done?

Reply:
I will try to implement a fix tomorrow. I don't think you need to generate `ydoc-server` with bazel now. Instead, ensure that the `.js` files are built using bazel, and then provide it to sbt via [environment variables](https://github.com/enso-org/enso/blob/6fa8d5f0b8d32a367ac65315f94a8bb884816a8e/project/BazelSupport.scala#L18-L23).

Reply:
> the `.js` files are built using bazel, and then provide it to sbt via [environment variables](https://github.com/enso-org/enso/blob/6fa8d5f0b8d32a367ac65315f94a8bb884816a8e/project/BazelSupport.scala#L18-L23).

Of course! I didn't even notice anyone else would suggest otherwise.